adoc/Release-Notes.adoc

```diff
@@ -33,20 +33,122 @@ endif::[]
+
+[id='sec.1_5_2.feature']
+==== Features and Fixes
+* Set `topologyKey` for `podAntiAffinity` rule to `kubernetes.io/hostname`
+* Inform operator of the need for CSR approval for Eirini internal registry in Helm notes
+* Pod restarts for many roles during UAA and SCF startup lessened for faster deployment
+* Expanded GoRouter ciphersuite to be compatible with a wider range of clients
+* Made Helm chart compatible with {k8s} 1.16
+* Use system cert store if `UAA_CA_CERT` is not specified
+* Includes these {cf} component versions:
+** app-autoscaler: 1.2.4
+** bits-services: 2.28.0
+** bpm: 1.1.6
+** capi: 1.89.0
+** cf-acceptance-tests: 12.17
+** cf-deployment: 12.17
+** cf-mysql: 36.15.0
+** cf-routing: 0.196.0
+** cf-sle12: 1.81.65
+** cf-smoke-tests: 40.0.123
+** cf-syslog-drain: 10.2.5
+** cf-usb: 1.0.1
+** cflinuxfs3: 0.153.0
+** credhub: 2.5.9
+** diego: 2.41.0
+** eirini: 0.0.26
+** garden-runc: 1.19.9
+** groot-btrfs: 1.0.5
+** loggregator: 105.6.3
+** loggregator-agent: 5.2.4
+** log-cache: 2.6.4
+** mapfs: 1.1.0
+** nats: 30
+** nfs-volume: 1.5.7
+** postgres-release: 26
+** pxc: 0.21.0
+** scf-helper: 1.0.13
+** sle15: 10.84
+** statsd-injector: 1.11.8
+** uaa: 74.12
+* Buildpacks:
+** binary-buildpack: 1.0.35
+** dotnetcore-buildpack: 2.3.0
+** go-buildpack: 1.9.4
+** java-buildpack: 4.27.0
+** nginx-buildpack: 1.1.2
+** nodejs-buildpack: 1.7.7
+** php-buildpack: 4.4.2
+** python-buildpack: 1.7.3
+** staticfile-buildpack: 1.5.2
+** ruby-buildpack: 1.8.3
+
+[id='sec.1_5_2.issue']
+==== Known Issues
+
+[IMPORTANT] 
+====
+* With {product} 1.5.2, the database seeder, UAA and credhub will use TLS by default to connect with an external database. Other SCF components will use the unencrypted TCP protocol. If you would like to configure the seeder, UAA & Credhub services to connect via insecure means in a trusted network, you will need to use the following parameters:
+
+`env.DB_EXTERNAL_SSL_MODE` - this is set to `true` by default and is used by the database seeder to communicate over TLS to the external database. Set it to `false` to have the seeder communicate over plain TCP.
+
+`env.UAADB_TLS` - this is set to `enabled` by default and is used by UAA to communicate over TLS to the external database. Set it to `disabled` to skip TLS for UAA connections to the external database. Please check the external database section in the documentation for other usages of this flag.
+
+`env.CREDHUB_DB_REQUIRE_TLS` - this is set to `true` by default and is used by credhub to communicate over TLS to the external database. Set it to `false` to disable TLS connections to the external database server from credhub. 
+
+Note that the external database server *must* support both TLS and non-encrypted connections; if the external database server only accepts TLS connections some SCF components will not be able to communicate with the database server.
```

Comment:
This only applies when TLS is enabled in the UAA + SCF parameters. A DB instance which doesn't support TLS will work when UAA + SCF are configured to use plaintext communication

Reply:
Sentence revised to indicate that it applies when TLS enabled.